src/screens/mentee-home/mentee-home.tsx

```diff
@@ -42,11 +42,19 @@ const MenteeHome: React.FC = () => {
     []
   );
 
+  const reloadOpportunities = useCallback(() => {
```

Comment:
I think It's better to rename all such functions via load
```suggestion
  const loadOpportunities = useCallback(() => {
```